DataFormats/Scouting/test/TestRun2ScoutingFormats.sh

```diff
@@ -59,19 +64,33 @@ cmsRun ${LOCAL_TEST_DIR}/test_readRun2Scouting_cfg.py || die "Failure using test
 # 10_2_0 was chosen because the 10_2_X release cycle was used
 # for 2018 data, and the scouting data formats were not updated
 # after 10_2_0.
-#
 
-file=testRun2Scouting_v3_v2_v2_v3_v2_v2_v2_CMSSW_8_0_7.root
+file=testRun2Scouting_v3_v2_v2_v3_v2_v2_v2_CMSSW_8_0_7_split_99.root
 inputfile=$(edmFileInPath DataFormats/Scouting/data/$file) || die "Failure edmFileInPath DataFormats/Scouting/data/$file" $?
 argsPassedToPython="--inputFile $inputfile --outputFileName testRun2Scouting2_CMSSW_8_0_7.root --muonVersion 2 --trackVersion 0 --vertexVersion 2"
 cmsRun ${LOCAL_TEST_DIR}/test_readRun2Scouting_cfg.py $argsPassedToPython || die "Failed to read old file $file" $?
 
-file=testRun2Scouting_v3_v2_v3_v3_v2_v2_v3_CMSSW_9_4_0.root
+file=testRun2Scouting_v3_v2_v2_v3_v2_v2_v2_CMSSW_8_0_7_split_0.root
```

Comment:
Would the for-loop pattern (over the splitlevel 0 and 99 files) be feasible here too?

Reply:
Done. The pairs with the same arguments are in a loop now. Looks better.